cmd/info.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright © 2021 Kaleido, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/hyperledger-labs/firefly-cli/internal/stacks"
+	"github.com/spf13/cobra"
+)
+
+// infoCmd represents the stop command
```

Comment:
should it be, "represents the info command"?

Reply:
Some bad copy pasta there. I'll fix. Thanks.